index.html

```diff
@@ -1,19 +1,140 @@
-    <title>Karma</title>
-    <link href='https://fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css'>
-    <link rel="stylesheet" href="css/normalize.css">
-    <link rel="stylesheet" href="css/style.css">
-    <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
-</head>
-<body>
-
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
-    <!-- All the images you need are in the 'img' folder -->
-
-</body>
-</html>
+    <!DOCTYPE html>
+    <html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta
+        name="viewport"
+        content="width=device-width, initial-scale=1, shrink-to-fit=no"
+        />
+        <title>Karma</title>
+        <link
+        href="https://fonts.googleapis.com/css?family=Roboto:400,500,300"
+        rel="stylesheet"
+        type="text/css"
+        />
+        <link rel="stylesheet" href="css/normalize.css" />
+        <link rel="stylesheet" href="css/style.css" />
+        <link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
+    </head>
+    <body>
+        <!-- Add your HTML markup here -->
+        <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
+        <!-- All the images you need are in the 'img' folder -->
+        <header>
+        <div class="header-container">
+            <div class="logo">
+            <img class="logo" src="./img/karma-logo.svg" alt="logo" />
+            </div>
+            <div class="hamburger">
+            <img class="hamburger-img" src="./img/menu-hamburger.svg" alt="hamburger button" />
+            </div>
+            <nav class="nav-bar">
+            <div class="nav-container">
+                <ul class="nav-btns">
+                <li><a href="./index.html" class="active">Meet Karma</a></li>
+                <li><a href="#" class="nav-link">How it Works</a></li>
+                <li><a href="#" class="nav-link">Store</a></li>
+                <li><a href="#" class="nav-link">Blog</a></li>
+                <li><a href="#" class="nav-link">Help</a></li>
+                <li><a href="#" class="nav-link">Login</a></li>
+                </ul>
+            </div>
+            </nav>
+        </div>
+        </header>
+        <main>
+        <section class="hero">
+            <div class="hero-container">
+            <div class="hero-title">
+                <h1>Introducing Karma</h1>
+            </div>
+            <div class="hero-content">
+                <p>Bring WiFi with you, everywhere you go.</p>
+            </div>
+            <div class="button-hero">
+                <button class="hero-btn" type="button">Learn More</button>
+            </div>
+            </div>
+        </section>
+        <section class="icons-section">
+            <div class="icon-container">
+            <div class="icon-title">
```

Comment:
the contents of the `div` with class `icons-container`, `hero-container` etc should be indented - this helps with readability as it makes it clear that these elements are children of their parent.